gpt4all-bindings/typescript/spec/chat-minimal.mjs

```diff
@@ -12,8 +11,6 @@ await createCompletion(
     "Why are bananas rather blue than bread at night sometimes?",
     {
         verbose: true,
+        nPredict: 10,
     }
-);
-await createCompletion(chat, "Are you sure?", {
-    verbose: true,
-});
+);
```

Comment:
Not a big deal, but I'd appreciate if you committed all text files with the trailing newline as per POSIX.